specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -4003,6 +4003,371 @@
+            "schema": {
+              "$ref": "#/definitions/TagResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request - Invalid Tags Definition | Invalid Tags Definition Properties."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Tags"
+        ],
+        "operationId": "Tags_ResourceGet",
+        "description": "Gets all the tags for the resource.",
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope of the resources.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The API version to use for the operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns tag information about the resource.",
+            "schema": {
+              "$ref": "#/definitions/TagResponse"
+            }
+          },
+          "404": {
+            "description": "Not Found - Resource Group Not Found."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets all the tags for a tracked resource.": {
+            "$ref": "./examples/GetTagsResource.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Tags"
+        ],
+        "operationId": "Tags_ResourceDelete",
+        "description": "Deletes all the tags for the resource.",
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope of the resources.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The API version to use for the operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Resources/tags/default": {
```

Comment:
Is this necessary? Should be covered by the above `{scope}` APIs, no?

Reply:
Merged into the {scope} one.